packages/react-core/src/demos/sampleData.tsx

```diff
@@ -0,0 +1,615 @@
+export const getRandomInteger = (min: number, max: number) => Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
Similar to above, we may not really need to export this.

Rather than creating a whole new file we also could just copy the `rows` data directly inside the `DataListPagination` file itself, outside the scope of the actual Demo export. Not sure if this is what @tlabaj had in mind or not, as this would at least give some reusable data within the react-core package; just depends how often we may use it and whether we want to clean it up as a followup at some point to be less "table-y".